packages/clay-navigation/src/Breadcrumb.tsx

```diff
@@ -0,0 +1,122 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import classNames from 'classnames';
+import React from 'react';
+import warning from 'warning';
+import {BreadcrumbEllipsis} from './BreadcrumbEllipsis';
+import {BreadcrumbItem, IBreadcrumbItem} from './BreadcrumbItem';
+
+interface IProps extends React.HTMLAttributes<HTMLOListElement> {
+	ellipsisBuffer?: number;
+
+	ellipsisProps?: Object;
+
+	items: Array<IBreadcrumbItem>;
+
+	spritemap: string;
+}
+
+const getBufferList = (
+	items: Array<IBreadcrumbItem>,
+	start: number,
+	end: number,
+	ellipsisProps: Object,
+	spritemap: string
+): (IBreadcrumbItem | React.ReactElement)[] => {
+	const removedItems = items.slice(start, Math.max(end, start));
+
+	return removedItems.length > 1
+		? [
+				<BreadcrumbEllipsis
+					items={removedItems}
+					key="breadcrumbEllipsis"
+					spritemap={spritemap}
+					{...ellipsisProps}
+				/>,
+		  ]
+		: removedItems;
+};
+
+const getBreadcrumbItems = (
```

Comment:
I see this and `getBufferList` is akin to the pagination rule, so I would choose to try to use the same function for the two and add some new params to customize some components... I think it would be best not to have rules duplicated in different places.

Reply:
Done!